### PR TITLE
Ensure websocket updates always contain the full processed data

### PR DESCRIPTION
**Pull request by bdraco**
When we called back the subscribers in fire_event we only
sent the changes. Sometimes this would be missing event_on
or event_ring_on if the websocket connected before the
integration setup the binary sensor. We now always return
the full processed data.

Should fix the last case of https://github.com/briis/unifiprotect/issues/181

Various cleanups (there were too many things called `data` to keep track of)

### CODE REVIEW

**Comment by bdraco** [comment]
Doing some double checking on my testing

**Comment by bdraco** [comment]
Looks good

File: pyunifiprotect/unifi_protect_server.py [hunk 1]
```diff
@@ -1137,8 +1141,8 @@ def fire_event(self, device_id, processed_event):
         self._update_device(device_id, processed_event)
 
         for subscriber in self._ws_subscriptions:
-            subscriber({device_id: processed_event})
+            subscriber({device_id: self._processed_data[device_id]})
```
Comment (bdraco) [inline pyunifiprotect/unifi_protect_server.py:1144]: This is the actual fix.

**Comment by briis** [comment]
Looks great, thanks.

**Pull request merged**